orderbook-rs/src/registry.rs

```diff
@@ -0,0 +1,14 @@
+use alloy_sol_types::sol;
+use ethers::contract::abigen;
+
+sol!(IOrderBookV3, "../out/IOrderBookV3.sol/IOrderBookV3.json");
+
+abigen!(IParserV1, "../out/IParserV1.sol/IParserV1.json");
+abigen!(
+    IExpressionDeployerV3,
+    r#"[
```

Comment:
magic strings should be constants if we can

Reply:
Macros do not accept &str constants as parameters.